pkgs/by-name/sh/shark/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  boost,
+  cmake,
+  fetchFromGitHub,
+  openssl,
+  stdenv,
+}:
+  stdenv.mkDerivation (finalAttrs: {
+    pname = "shark";
+    version = "4.1.0";
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "shark";
  version = "4.1.0";
```
please also fix the remaining file

---

Thanks @SuperSandro2000 ! 

I have used `nixpkgs-fmt` for formatting all the files.

---

We nowdays use nixfmt-rfc-style for new files.